### DISCUSSION TITLE
search/replace  drag to other position on screen

### DISCUSSION THREAD

**Discussion by ldijkman** [ajaxorg/ace#5325] [General]
search/replace  drag to other position on screen

is it possible?

https://ldijkman.github.io/Ace-Scrollbars/Ace_editor_custom_scrollbar.html

https://ldijkman.github.io/Ace_MiniMap/ace_editor_minimap.html

![2023-09-24-081719_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/51d758ac-3cbc-4a71-be69-ab016f5a56b9)

![2023-09-24-081621_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/3319aacf-d1f1-4c40-af1f-a6adae294730)

**Comment by ldijkman** [comment]
ok i can move it a bit
but the right side keeps sticking to the right

video showing move search replace https://youtu.be/9rnnVYyaO0s

video demo source https://plnkr.co/edit/EwKPML568gRaCR53

if i uncheck right in chrome inspect right does not stick to right
``` 
.ace_search {
    border-radius: 0px 0px 0px 5px;
    border-right: 0 none;
    right: 0;
    cursor: move;
}
```

placed some jquery in the editor on change
think it should be run after the search popup creation

```
 $( ".ace_search" ).draggable();  // after an edit the search / replace popup is draggable 
```


![2023-09-24-110516_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/5d491769-3ac0-436b-bd75-11fbd872dc2c)

**Reply by ArunPatal** [reply]
Hi, minimap look really nice. Would like to see it as a part of extension. Or is it already?

**Reply by ldijkman** [reply]
Hello 
I do not have the knowledge how to make it an extension
I am a welder mechanic playing a bit with code
Input is apreciated

Ace Seventh Heaven
https://ldijkman.github.io/Ace_Seventh_Heaven/Seventh_Heaven.html

**Comment by ldijkman** [comment]
Got it

https://ldijkman.github.io/Ace_MiniMap/ace_editor_minimap.html

![Seventh_Heaven_Ace_Editor_move_drag_search_replace](https://github.com/ajaxorg/ace/assets/45427770/2e870e74-303c-4925-a9e9-ad773a925086)
